apps/tup-cms/Dockerfile

```diff
@@ -1,5 +1,5 @@
-# TACC/Core-CMS#v3.12.0-beta.3
-FROM taccwma/core-cms:0992f47
+# TACC/Core-CMS#707 (v3.12.0-beta.4 candidate)
+FROM taccwma/core-cms:1f7b95d
```

Comment:
Though there have been changes since [`1f7b95d` in TACC/Core-CMS#707](https://github.com/TACC/Core-CMS/pull/707/commits/1f7b95d), those changes[^1] an inconsequential to https://github.com/TACC/tup-ui/. Let's keep this commit has, because we tested it.

[^1]: The changes are (fixes) relevant to other projects that use https://github.com/TACC/Core-CMS.